Compiler warning in `cctest/inspector/test_network_requests_buffer.cc`

```
In file included from ../../test/cctest/inspector/test_network_requests_buffer.cc:1:
../../deps/googletest/include/gtest/gtest.h:1394:11: warning: comparison of integers of different signs: 'const unsigned long' and 'const int' [-Wsign-compare]
 1394 |   if (lhs == rhs) {
      |       ~~~ ^  ~~~
../../deps/googletest/include/gtest/gtest.h:1413:12: note: in instantiation of function template specialization 'testing::internal::CmpHelperEQ<unsigned long, int>' requested here
 1413 |     return CmpHelperEQ(lhs_expression, rhs_expression, lhs, rhs);
      |            ^
../../test/cctest/inspector/test_network_requests_buffer.cc:171:5: note: in instantiation of function template specialization 'testing::internal::EqHelper::Compare<unsigned long, int, nullptr>' requested here
  171 |     EXPECT_EQ(buffer.total_buffer_size(), 15);
      |     ^
../../deps/googletest/include/gtest/gtest.h:1888:54: note: expanded from macro 'EXPECT_EQ'
 1888 |   EXPECT_PRED_FORMAT2(::testing::internal::EqHelper::Compare, val1, val2)
      |                                                      ^
1 warning generated.
```

--- a/test/cctest/inspector/test_network_requests_buffer.cc
+++ b/test/cctest/inspector/test_network_requests_buffer.cc
@@ -168,7 +168,7 @@ TEST(RequestsBuffer, EnforceLimit) {
     buffer.find("4")->second.push_request_data_blob(
         protocol::Binary::fromSpan(kDummyData, 10));
     // "3" and "4" are kept.
-    EXPECT_EQ(buffer.total_buffer_size(), 15);
+    EXPECT_EQ(buffer.total_buffer_size(), 15u);
   }
 
   EXPECT_FALSE(buffer.contains("1"));

